synthtool/languages/node.py

```diff
@@ -258,6 +259,20 @@ def postprocess_gapic_library_hermetic(hide_output=False):
     logger.debug("Post-processing completed")
 
 
+def collect_versions_from_src(default_version: str) -> List[str]:
```

Comment:
This is close to the same behave as `def detect_versions(path="./src") -> List[str]:`, which was added in an earlier iteration of the processor to solve this problem.

Can we remove `detect_version`?

Reply:
Actually, I'll fix detect_version instead.().